server/src/main/java/io/deephaven/server/table/ops/UpdateByGrpcImpl.java

```diff
@@ -187,7 +198,30 @@ private static EmaSpec adaptEma(UpdateByEma ema) {
                 : EmaSpec.of(adaptTimescale(ema.getTimescale()));
     }
 
-    private static OperationControl adaptEmaOptions(UpdateByEma.UpdateByEmaOptions options) {
+    private static EmsSpec adaptEms(UpdateByEms ems) {
+        return ems.hasOptions() ? EmsSpec.of(adaptEmaOptions(ems.getOptions()), adaptTimescale(ems.getTimescale()))
+                : EmsSpec.of(adaptTimescale(ems.getTimescale()));
+    }
```

Comment:
What this is is not self evident from the name.  Is this implementing EMAs or doing something else?

Reply:
`Ems` is shorthand for Exponential Moving Sum.  To recap, the EM operators are `Ema` (average), `Ems` (sum), `EmMin` (minimum), `EmMax` (maximum), `EmStd` (standard deviation).

Reply:
Added comments to distinguish functions.